subprojects/gradle/instrumentation-test-impact-analysis/build.gradle.kts

```diff
@@ -12,8 +13,8 @@ val kotsonVersion: String by project
 
 dependencies {
     implementation(project(":subprojects:gradle:android"))
+    implementation(project(":subprojects:gradle:impact-shared"))
```

Comment:
Can you please describe changes in a project layout?
It looks like a new convention or a type of module.
Is it just a shared code between gradle plugins or something more complex?

Reply:
i had to rename `impact` -> `impact-plugin` because of publication quirks
and `impact` to something else
that's the only reason